src/common/transformations/src/transformations/common_optimizations/adaptive_pool_to_reduce.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/adaptive_pool_to_reduce.hpp"
+
+#include <memory>
+#include <vector>
+
+#include "itt.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/opsets/opset11.hpp"
```

Comment:
please use the concrete ops versions everywhere, e.g.
ov::op::v1 instead of opset11
please delete opset11 include from the file
We have an issue with the transformation static lib size on Win platform. 
We need to double check debug build on Win platform before these changes are merged.